src/main/kotlin/org/jetbrains/research/ml/codetracker/server/PluginServer.kt

```diff
@@ -1,17 +1,28 @@
 package org.jetbrains.research.ml.codetracker.server
 
 import kotlinx.serialization.builtins.serializer
+import org.jetbrains.research.ml.codetracker.models.Country
+import org.jetbrains.research.ml.codetracker.models.Gender
+import org.jetbrains.research.ml.codetracker.models.PaneLanguage
 import org.jetbrains.research.ml.codetracker.models.Task
 
 object PluginServer {
     // Todo: get genders, countries and experiences
 
-    fun getAvailableLanguages(): List<String> {
-        return CollectionsQueryExecutor.getCollection("language/all", String.serializer())
+    fun getAvailableLanguages(): List<PaneLanguage> {
+        return CollectionsQueryExecutor.getCollection("language/all", PaneLanguage.serializer())
     }
 
     fun getTasks(): List<Task> {
         return CollectionsQueryExecutor.getCollection("task/all", Task.serializer())
     }
 
+    fun getGenders(): List<Gender> {
+        return CollectionsQueryExecutor.getCollection("gender/all", Gender.serializer())
+    }
+
+    fun getCountries(): List<Country> {
+        return CollectionsQueryExecutor.getCollection("country/all", Country.serializer())
```

Comment:
Shouldn't we include urls into models as field?
for example:
data class Task(...) {
val URL = "task/all"
}

also, maybe it makes sense to create an interface for Model with abstract variable URL? and make all current models inherit from it